### DISCUSSION TITLE
Opening a relative Link to an DOCX doesn't work

### DISCUSSION THREAD

**Discussion by hilavoku** [sumatrapdfreader/sumatrapdf#3457] [Q&A]
Hello.

I searched the forum to see if anyone else had the same problem, but I didn't find anything similar.

I have a PDF file that contains a relative link to a Word DOCX file.
When I open the PDF file and click on the link in SumatraPDF, I get the error message "Error loading C:\Applications\SumatraPDF\Test-PDF.docx".
If I change the UseTabs = true setting in SumatraPDF-settings.txt to UseTabs = false and then click on the link, the behavior is now different. SumatraPDF opens another instance of SumatraPDF, which displays the same error message, but immediately opens the DOCX document with Microsoft Word. 

Very strange behavior. I have tried all the different SafeFileTypes settings in sumatrapdfrestrict.ini, but no change.

Has anyone else had the same observations?

When I open the PDF with FoxitReader and click on the link, after a confirmation prompt, the Word document opens immediately.

Thanks in advance for any tips.

**Comment by hilavoku** [comment]
Forgot to mention that I use Version 3.5 of SumatraPDF.

**Comment by GitHubRulesOK** [comment]
Hmm in theory SumatraPDF should NOT run such attachments as could be viral however looking at [test file](https://github.com/sumatrapdfreader/sumatrapdf/files/4527465/test_pdf.pdf) from https://github.com/sumatrapdfreader/sumatrapdf/issues/1569 

@kjk  something is not right with the current link description 
![image](https://user-images.githubusercontent.com/12754913/233796444-1ba5d760-3e06-4f47-b888-9ae453de9dac.png)
but resolves to correct name for save
![image](https://user-images.githubusercontent.com/12754913/233796669-f5b4cd97-dc8e-4f0b-a7d9-f54eaeadc95d.png)
![image](https://user-images.githubusercontent.com/12754913/233796753-53f74fa1-6e66-4f54-a26f-bbdff0c202e5.png)

So adding a relative link to that file, I get similar error even though the file is now there ./BookTest2.xls

![image](https://user-images.githubusercontent.com/12754913/233798587-3af70d9b-aa81-45be-bbe9-6a5b3fc6339d.png)

switching tabs off does generate odd behaviour

![image](https://user-images.githubusercontent.com/12754913/233798801-1f418935-4b0d-4aa6-8578-6be6579fabe2.png)

Multiple clicking opens a file that seems to be randomly chosen here is that modified text  file 
[test_pdf.pdf](https://github.com/sumatrapdfreader/sumatrapdf/files/11301965/test_pdf.pdf)


![image](https://user-images.githubusercontent.com/12754913/233798954-f7f986fa-4519-43ad-a21e-e1832cde2052.png)

Comparison with how other apps handle the click
![image](https://user-images.githubusercontent.com/12754913/233799167-ba2519d7-cc43-4fef-b590-81cf53718fcd.png)

Ideally
---
There should be a message explaining the blocker
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/72396db8-fa8b-4b25-a0a8-085441ba2289)

**Comment by hilavoku** [comment]
Here is a Example. Attached you find a Word Document and the Created PDF from it. Try to open the Word File from the PDF File. It will throw an error.
[Simple.docx](https://github.com/sumatrapdfreader/sumatrapdf/files/11301937/Simple.docx)
[Simple.pdf](https://github.com/sumatrapdfreader/sumatrapdf/files/11301938/Simple.pdf)

**Comment by hilavoku** [comment]
It seems that no one wants to open Word or Excel documents from a PDF.
It is obvious that there is a small bug in the behavior. Sumatra only opens other documents via link if tabs are not activated in the settings file.

**Reply by MesaBul** [reply]
I have the same problem. Did you fix it?

I'm trying to open remote file but SumatraPDF always open from C:/path, i can't open it like J:/path for example.

When the path is okay, doesn't show message but don't open nothing, i tried to put tabs = false but nothing change.

**Reply by GitHubRulesOK** [reply]
@MesaBul 
This question was about running external unsafe files
Your issue is different since although PDF is an unsafe external file, there is a concession for opening a companion.pdf as part of a group of pdfs
